Source/Urho3D/Resource/DecompressETC2.h

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
The header and implementation files are missing the Urho3D copyright notice. Also, just for consistency sake, I think all the newly added functions should be declared using `Urho3D` namespace.